### PR TITLE
Allow failures in chrome stability check

### PR DESCRIPTION
**Pull request by jgraham**
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5624)

### CODE REVIEW

File: test/chrome_unstable.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<meta name=timeout content=long>
+<title>Example test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(function() {
+  if (navigator.userAgent.indexOf("Chrome") > -1 && Math.random() > 0.5) {
+    assert_true(false);
```
Comment (jugglinmike) [inline test/chrome_unstable.html]: This is my new favorite test.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/224003415)

# Lint #

Passed

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/224003417)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 8c80c5c005993f34ccfd6433a5b361fecd84166d
Using browser at version BuildID 20170420100256; SourceStamp 27311156637f9b5d4504373967e01c4241902ae7
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5624/test/chrome_unstable.html">/test/chrome_unstable.html</a></summary>


|    Subtest     | Results | Messages |
|----------------|---------|----------|
|                | OK      |          |
| `Example test` | PASS    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/224003423)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 8c80c5c005993f34ccfd6433a5b361fecd84166d
Using browser at version 59.0.3071.9 dev
Starting 10 test iterations
## Unstable results ##


|             Test             |    Subtest     |          Results           |                Messages                |
|------------------------------|----------------|----------------------------|----------------------------------------|
| `/test/chrome_unstable.html` | `Example test` | **FAIL: 7/10, PASS: 3/10** | `assert_true: expected true got false` |

## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5624/test/chrome_unstable.html">/test/chrome_unstable.html</a></summary>


|    Subtest     |          Results           |                Messages                |
|----------------|----------------------------|----------------------------------------|
|                | OK                         |                                        |
| `Example test` | **FAIL: 7/10, PASS: 3/10** | `assert_true: expected true got false` |

</details>
</details>

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5624/)

**Pull request merged**